html/changelogs/.all_changelog.yml

```diff
@@ -41373,209 +41373,22 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   zxaber:
   - tweak: The NtOS home screen has been updated with disk and ID eject buttons.
 2020-08-02:
-  ArcaneMusic:
```

Comment:
Disable the actions in your repository to avoid this kind of thing.
And revert the changelog compile commits to easily undo these changes.